src/plugins/vis_type_timeseries/server/lib/vis_data/annotations/get_request_params.ts

```diff
@@ -6,25 +6,38 @@
  * Side Public License, v 1.
  */
 
+import { AnnotationItemsSchema, PanelSchema } from 'src/plugins/vis_type_timeseries/common/types';
 import { buildAnnotationRequest } from './build_request_body';
-import { getEsShardTimeout } from '../helpers/get_es_shard_timeout';
 import { getIndexPatternObject } from '../../search_strategies/lib/get_index_pattern';
+import {
+  VisTypeTimeseriesRequestHandlerContext,
+  VisTypeTimeseriesRequestServices,
+  VisTypeTimeseriesVisDataRequest,
+} from '../../../types';
+import { AbstractSearchStrategy } from '../../search_strategies';
+
+export type AnnotationServices = VisTypeTimeseriesRequestServices & {
+  capabilities: any;
```

Comment:
These are the `DefaultSearchCapabilities`? Can't we type it?

Reply:
thanks, typed